cmd/helm/pull_test.go

```diff
@@ -127,9 +145,27 @@ func TestPullCmd(t *testing.T) {
 				filepath.Join(outdir, "repositories.yaml"),
 				outdir,
 			)
+			// Create file or Dir before helm pull --untar, see: https://github.com/helm/helm/issues/7182
+			if tt.existFile != "" {
+				file := filepath.Join(outdir, tt.existFile)
+				_, err := os.Create(file)
+				if err != nil {
+					t.Fatal("err")
+				}
+			}
+			if tt.existDir != "" {
+				file := filepath.Join(outdir, tt.existDir)
+				err := os.Mkdir(file, 0755)
+				if err != nil {
+					t.Fatal(err)
+				}
+			}
 			_, out, err := executeActionCommand(cmd)
 			if err != nil {
 				if tt.wantError {
+					if tt.wantErrorMsg != "" && strings.Contains(tt.wantErrorMsg, err.Error()) {
```

Comment:
Looks like in this case, the error message is something that goes out to the user in the console output and we want it to be informative and correct for better UX. 

And in regards to that, I think we could do an exact check on the error message compared to a `Contains` check. This will also catch issues that @hickeyma pointed out 😄 What do you think @zwwhdls ?

Reply:
I don't think it's necessary.
But if you insist on this, I will fix it too.